docker_run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+###  SET HOST FILE PATHS HERE. All paths must be absolute!  ###
+HOST_CFG=$(pwd)/rabbitmq-worker.conf
+HOST_LOG=$(pwd)/rabbitmq-worker.log
+HOST_ERR=$(pwd)/rabbitmq-worker.err
```

Comment:
From what I know of docker, and this official doc https://docs.docker.com/engine/admin/logging/view_container_logs/, it seems you are better off redirecting to STDOUT and STDERR instead.